frontend/packages/api_client/src/apis/V2Api.ts

```diff
@@ -463,6 +467,45 @@ export class V2Api extends runtime.BaseAPI {
         return await response.value();
     }
 
+    /**
+     * Send a POST to action image/ with the file in the `logo` or `favicon` field to add or update it.
+     * Endpoint for saving favicon or logo images
+     */
+    async instancesOpenedxConfigImageRaw(requestParameters: InstancesOpenedxConfigImageRequest): Promise<runtime.ApiResponse<object>> {
+        if (requestParameters.id === null || requestParameters.id === undefined) {
+            throw new runtime.RequiredError('id','Required parameter requestParameters.id was null or undefined when calling instancesOpenedxConfigImage.');
+        }
+
+        const queryParameters: runtime.HTTPQuery = {};
+
+        const headerParameters: runtime.HTTPHeaders = {};
+
+        if (this.configuration && this.configuration.apiKey) {
+            headerParameters["Authorization"] = this.configuration.apiKey("Authorization"); // api_key authentication
+        }
+
+        if (this.configuration && (this.configuration.username !== undefined || this.configuration.password !== undefined)) {
```

Comment:
I'd use instead:

if (this.configuration) {
    if (this.configuration.apiKey) {}
    else if (this.configuration.username !== undefined || this.configuration.password !== undefined) {}
    else { 
        // handle error
    }
}

Reply:
I can't edit these files, they are auto-generated by the DRF YASG.

Reply:
Never mind then.